ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -21,6 +21,7 @@ const (
 	JobStatusStopped   JobStatus = "STOPPED"
 	JobStatusSucceeded JobStatus = "SUCCEEDED"
 	JobStatusFailed    JobStatus = "FAILED"
+	JobStatusScheduled JobStatus = "SCHEDULED"
```

Comment:
See the link on line 14 (https://docs.ray.io/en/latest/cluster/running-applications/job-submission/doc/ray.job_submission.JobStatus.html#ray.job_submission.JobStatus). JobStatus is for mirroring job status from the Ray job API. So it doesn't make sense to set this status to SCHEDULED. In the case where a JobDeploymentStatus is set to `Scheduled` the JobStatus can be JobStatusNew